UserTools/LoadWCSim/wcsimT.h

```diff
@@ -47,7 +47,7 @@ public :
    WCSimRootOptions   *wcsimrootopts=nullptr;
    
    // TODO implement a verbosity arg
-   int             verbose=1;
+   int             verbose=5;
```

Comment:
there's no point changing this as both constructors will set the verbosity, so this value will always be overwritten.

Reply:
it's also bad practice to set default verbosities so high. Generally they should set to only print errors and warnings.

Reply:
feels like none of the changes in this file should be committed....